src/GazeboThruster.cpp

```diff
@@ -192,6 +192,7 @@ void GazeboThruster::updateBegin(common::UpdateInfo const& info)
             thruster != thrusters.end(); ++thruster)
     {
         physics::LinkPtr link = model->GetLink( thruster->name );
+        link->AddLinkForce( thruster->effort*gazebo::math::Vector3::UnitX);
         link->AddLinkForce( thruster->effort*thruster->direction, thruster->position );
```

Comment:
> Direction is not UnitX and the norm may not be 1. Position is not the (0,0,0) of thruster frame.
For Ma=0 => direction=0, and just the original force is considered.

Got it now ... Given that position and direction are not position and direction, **give them better names**. Like `added_mass_compensation_direction` and `added_mass_compensation_position`. `thruster->direction` _can only be the thruster direction_